test/FunctionsV2/TestCleanup.cs

```diff
@@ -0,0 +1,89 @@
+﻿using DurableTask.AzureStorage;
+using FluentAssertions.Common;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace WebJobs.Extensions.DurableTask.Tests.V2
+{
+    public class TestCleanup
+    {
+        private readonly ITestOutputHelper output;
+
+        private readonly TestLoggerProvider loggerProvider;
+
+        public TestCleanup(ITestOutputHelper output)
+        {
+            this.output = output;
+            this.loggerProvider = new TestLoggerProvider(output);
+        }
+
+        // This test should never fail. The sole purpose is to cleanup old taskhubs in the CI
+        // storage account to prevent clutter now that TaskHub names are non-deterministic.
+        [Fact]
+        public async Task CleanupOldAzureStorageTaskHubs()
```

Comment:
Do we have a way to make sure this is run before any other tests? I'm asking because not all tests are guaranteed to run on a CI execution (for example, if another test fails first, we might not do a clean-up step), so it would be good to perform the clean-up as eagerly / early as possible.

Reply:
It's not super important that that this runs every test run, as it should clean up more than the number of task hubs that it generates, and `clean-up` isn't that important. This is meant to just prevent us from ballooning 500+ taskhubs on every CI run without ever cleaning up. 

I was debating not putting this as a test and having a timer trigger perform this daily at midnight (with no limit on the number of taskhubs it deletes), but that is just more infrastructure to maintain.

Reply:
Understood and agreed, we can leave this as a "test", _for now_ ;)